x-pack/test/alerting_api_integration/security_and_spaces/tests/actions/builtin_action_types/jira.ts

```diff
@@ -123,7 +126,8 @@ export default function jiraTest({ getService }: FtrProviderContext) {
           config: {
             apiUrl: jiraSimulatorURL,
             projectKey: mockJira.config.projectKey,
-            casesConfiguration: mockJira.config.casesConfiguration,
+            incidentConfiguration: mockJira.config.incidentConfiguration,
```

Comment:
regarding the rename of `casesConfiguration` - since actions created with the old config value could already exist from the previous release, should probably have a test for that.  Might be hard to test for.  Especially hard since the field is part of AAD so you can't just overwrite the field with the generic saved object API.  Probably a case for es-archiver, sorry to say.  another one that @mikecote may have some input on

Reply:
++ es archiver is probably best in this scenario but there may be issues where it erases other data the test suite relies on (users  / spaces). @gmmorris is going through the exact same issue in the RBAC work, maybe a change he's working on will help here?

Reply:
I think based on my note above: https://github.com/elastic/kibana/pull/73778#discussion_r481401109. I'd be +1 on reverting it back to `casesConfiguration` since we know the field is going away soon and we don't have to worry about a migration or tests to cover this.